app/src/common/shared/com/igalia/wolvic/browser/engine/Session.java

```diff
@@ -74,6 +75,7 @@ public class Session implements WContentBlocking.Delegate, WSession.NavigationDe
     private static UriOverride sUserAgentOverride;
     private static UriOverride sDesktopModeOverrides;
     private static final long KEEP_ALIVE_DURATION_MS = 1000; // 1 second.
+    private static final String DEFAULT_USER_AGENT = "Mozilla/5.0 (Android 10; Mobile VR; rv:105.0) Gecko/105.0 Wolvic/" + BuildConfig.VERSION_NAME;
```

Comment:
I don't think we can do this, I'd bet we need to include also the AppleWebKit and Chromium parts of the user agent, otherwise I'm afraid many sites will be broken.